superset/utils/dashboard_import_export.py

```diff
@@ -38,12 +38,17 @@ def import_dashboards(session, data_stream, import_time=None):
     session.commit()
 
 
-def export_dashboards(session):
+def export_dashboards(session, dashboard_id=None, dashboard_title=None):
     """Returns all dashboards metadata as a json dump"""
     logging.info('Starting export')
     dashboards = session.query(Dashboard)
     dashboard_ids = []
     for dashboard in dashboards:
-        dashboard_ids.append(dashboard.id)
-    data = Dashboard.export_dashboards(dashboard_ids)
+        if dashboard_id or dashboard_title:
+            if dashboard.id == dashboard_id or \
+                dashboard.dashboard_title == dashboard_title:
+                dashboard_ids.append(dashboard.id)
+        else:
+            dashboard_ids.append(dashboard.id)
+    data = Dashboard.export_dashboards(dashboard_ids) if dashboard_ids else {}
```

Comment:
could be good to:
```python
dashboard_ids = [d.id for d in query.all()]
if not dashboard_ids:
    click.secho('No dashboard found', fg='red)
    sys.exit(1)
```

Reply:
I've made it work this way and have added a `superset.exceptions.DashboardNotFoundException` that `dashboard_import_export.export_dashboards()` raises and the `cli.py` caller intercepts so he can use `click.echo(click.style(message, fg='red'))`. It does not warn about a single incorrect id/title among multiple id/title arguments. I could add that as well if you would like.